app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
+    @user = User.find(params[:user_id])
+    @post = @user.posts.find(params[:post_id])
+    @comment = @post.comments.build(comment_params.merge(user_id: current_user.id))
+
+    if @comment.save
+      redirect_to user_post_path(@user, @post)
+    else
+      render 'new'
+    end
+  end
```

Comment:
- [ ] Nice work creating the `create` action for the Comment Controller :clap: However, according to the project requirement the comment should be created on behalf of the currently logged-in user (current_user). Therefore I suggest you make use of the `current_user` method that you created in the Application controller (`Ex: @user = current_user`)

- [ ] Kindly add the `new` action to the Comment Controller. It will handle the creation of a new comment